src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1656,20 +1655,13 @@ public void testCachedFile() throws Exception {
         checker.destroy();
     }
 
-    @SuppressForbidden
     @Test
     public void testUnmappableCharacters() throws Exception {
         final String[] expected = {
-            "4: " + getCheckMessage(LineLengthCheck.class, MSG_KEY, 75, 238),
+            "14: " + getCheckMessage(LineLengthCheck.class, MSG_KEY, 80, 100),
         };
-
-        final DefaultConfiguration checkConfig = createModuleConfig(LineLengthCheck.class);
-        checkConfig.addProperty("max", "75");
-
-        final DefaultConfiguration checkerConfig = createRootConfig(checkConfig);
-        checkerConfig.addProperty("charset", "IBM1098");
-
-        verify(checkerConfig, getPath("InputCheckerTestCharset.java"), expected);
+    
+        verifyWithInlineXmlConfig(getPath("InputCheckerTestCharset.xml"), expected);
```

Comment:
` tests did not pass without mutation when calculating line coverage. Mutation testing requires a green suite.` 
 from https://github.com/checkstyle/checkstyle/actions/runs/8565411036/job/23473432315?pr=14762

this test is failing. the file name should have  `.java` extension not  `.xml` and the violation line 13 not 14 make sure test suite is green after you make any changes